### ISSUE TITLE
Popup configuration "qgis" for children not working

### ISSUE THREAD

**Issue by itsakifa** [3liz/lizmap-plugin#326]
I would like to show HTML-styled children popups (without geometry) below the parents havig a "qgis"-configured popup with expressions and links. 
At the moment, it is not possible to chose "qgis" through the Lizmap plugin for layers without geometry.
Also, when parents have "qgis" popup configuration, no children are displayed.
And lastly, children are not displayed (in any popup configuration) when coming from a view in the database rather than a table.

I'm using QGIS 3.10 and Lizmap 3.3

**Comment by aeduard** [comment]
I cannot reproduce any of your issues. I'm using qgis type popups with child layers (1:N relation). I can select qgis popup configuration for non-geometry layer. Also views are working, but they need to have columns with unique values.

I'm using latest lizmap plugin (3.3.2), with lizmap web client 3.3.10 and qgis server 3.10. Also my data are coming from postgres db.

**Comment by Gustry** [comment]
> I can select qgis popup configuration for non-geometry layer.

Hum, this weird indeed.
I just tried indeed : 

![Screenshot from 2020-12-29 18-01-43](https://user-images.githubusercontent.com/1609292/103279280-b5abf180-49cd-11eb-8cc7-3d3846ab7640.png)

While I can see in the code that there is an explicit rule : https://github.com/3liz/lizmap-plugin/commit/e0278f998b5605b872fec85f2d42069f83673bf8

**Comment by Gustry** [comment]
Ah yes, it's already enabled if QGIS > 3.10 : https://github.com/3liz/lizmap-plugin/commit/d609bf35f0f172191cef4468d78f5cece5814558

Sorry

So I'm not sure what is missing @itsakifa ?

**Comment by itsakifa** [comment]
Thanks for your answers @Gustry and @aeduard. I finally resolved my issues - due to several missing configs on my side:
- my children table did not contain any unique key. as it is without geometry, I had to specify the unique key after import in the qgis project via Properties > Update of the SQL layer
- I had to make sure the children table is named with the original name of the table in the postgres database. When changing the name, the children table and the correct relations to the parent layer were still displayed through the Lizmap attribute table, but the popup did not work ("layer not found" in the logs). 
I still had to manually change the popup layer style to "qgis" for the children table without geometry.
